### PR TITLE
feat(TDP-1465): updated flags to allow for override colours in articl…

### PR DESCRIPTION
**Pull request by adamosborne-tnl**
…e templates such as IN_DEPTH

### CODE REVIEW

File: packages/ts-components/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "lint": "tslint --project . && prettier \"src/**/*.{ts,tsx}\" --list-different",
     "test:lint": "tslint --project . && prettier \"src/**/*.{ts,tsx}\" --list-different",
     "test:unit": "jest --coverage",
+    "watch:build": "run-s clean && run-p build:* \"build:module -- -w\"",
```
Comment (themischatzis) [inline packages/ts-components/package.json:32]: Can we add a description of what this command is doing in the README please?

File: packages/ts-components/src/components/article-flag/ArticleFlag.tsx [hunk 2]
```diff
@@ -27,51 +27,58 @@ const ArticleFlag: React.FC<{ color: string; title: string }> = ({
...
   ]);
+};
 
 export type FlagType = Array<{
   expiryTime: string | null;
   type: string;
 }>;
 
-const FlagsView: React.FC<{ allFlags: FlagType }> = ({ allFlags }) => {
+const FlagsView: React.FC<{ allFlags: FlagType; overridecolor?: string }> = ({
```
Comment (themischatzis) [inline packages/ts-components/src/components/article-flag/ArticleFlag.tsx:73]: Can we replace the `overridecolor` to be with in lowerCamelCase like `overrideColor`?

**Review by themischatzis — COMMENTED** [review]
Overall looks great! I just left a couple of suggestions.

File: packages/ts-components/package.json (hunk 1 above)
Comment (adamosborne-tnl) [reply packages/ts-components/package.json:32]: ok. done

File: packages/ts-components/src/components/article-flag/ArticleFlag.tsx (hunk 2 above)
Comment (adamosborne-tnl) [reply packages/ts-components/src/components/article-flag/ArticleFlag.tsx]: yup, done, can't believe i didn't do that in the first place 😱

**Review by themischatzis — APPROVED** [review]

File: packages/ts-components/README.md [hunk 3]
```diff
@@ -20,5 +20,5 @@ All the build steps have been updated to handle all the build steps:
 
 ## Storybook
 
-For now, you will need to do a `yarn build` for storybook to update.
+For now, you will need to do a `yarn build` for storybook to update, or you could run `yarn watch:build` to update storybook on save
```
Comment (bibicollins1) [inline packages/ts-components/README.md:23]: 🍾 🍾

**Review by bibicollins1 — APPROVED** [review]

**Pull request merged**